src/fullcalendar/localization/dateFormattingConfig.js

```diff
@@ -42,6 +42,9 @@ const getDateFormattingConfig = () => {
 			},
 			timeGridDay: defaultConfig,
 			timeGridWeek: defaultConfig,
+			listMonth: {
+				listDayFormat: 'LL, DDDD',
```

Comment:
```suggestion
				listDayFormat: 'LL, dddd',
				listDaySideFormat: false,
```

Change 1: DDDD is the day of year in numerical format (e.g. `001 002 ... 364 365`).
Change 2: Disable the right side because the day is now shown on the left side already.

Reply:
Hey. Thank you for your suggestions. I've changed the files accordingly. Kindly acknowledge. Please let me know if any further changes are required.